src/frontend/src/components/pinInput.ts

```diff
@@ -207,15 +207,16 @@ const onPaste = ({
     return;
   }
 
-  // Actually set the value (for reasons unclear, preventDefault() is not necessary here as tested on
-  // Chrome, Safari)
+  // Set the values manually
   for (const [input, char] of toBePasted) {
     input.value = char;
   }
+  // Prevent actually pasting the value in any of the fields since we just did this manually
+  event.preventDefault();
```

Comment:
Note, `event` seems to be a reserved key if I get https://www.w3schools.com/js/js_reserved.asp correctly.
I generally use `$event` to avoid any unexpected issue even if transpilled but not sure it's relevant here.

Reply:
Thanks, I think you've mentioned this before (might have been about `error`?). I'll do a batch replace soon.